exchanges/request-policy/src/requestPolicyExchange.ts

```diff
@@ -9,10 +9,32 @@ import { pipe, tap, map } from 'wonka';
 const defaultTTL = 5 * 60 * 1000;
 
 export interface Options {
+  /** A function allowing you to return a boolean on whether or not to upgrade the
+   * operation to "cache-and-network". */
   shouldUpgrade?: (op: Operation) => boolean;
+  /** The TTL in ms for which we should not upgrade an operation for. */
   ttl?: number;
 }
 
+/** Creates an `Exchange` that can upgrade the request-policy to "cache-and-network" of an urql query operation.
```

Comment:
```suggestion
/** Exchange factory that upgrades request policies to `cache-and-network` for queries outside of a defined `ttl`.
```

Reply:
^that's just for consistency and phrasing (i.e. can upgrade -> upgrades; urql query operation -> queries; creates an exchange -> exchange factory)